src/Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices/Authentication/CookieExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.Owin.Security.Cookies;
+using Microsoft.Owin.Security.Interop;
+using Owin;
+
+namespace Microsoft.Owin.Security.Cookies;
+
+public static class CookieExtensions
+{
+    public static CookieAuthenticationOptions ConfigureSharedCookie(this IAppBuilder app, string sharedApplicationName,
```

Comment:
This should be an extension method on CookieAuthenticationOptions, not IAppBuilder.